common/utils/utility-types.ts

```diff
@@ -24,3 +24,13 @@ export type OptionalToUndefined<T> = {
 export type ElementFromComponent<C extends FunctionComponent> = ReactElement<
   ComponentProps<C>
 >;
+
+/** Allows you to set a string type that requires a certain prefix
```

Comment:
This is cool

Reply:
It's the reason for the typescript bump.